packages/tradershub/src/features/cfd/modals/ButtonGroups/AddAccountButtonsGroup.tsx

```diff
@@ -12,12 +12,12 @@ type TAddAccountButtonsGroupProps = {
 const AddAccountButtonsGroup = ({ password }: TAddAccountButtonsGroupProps) => {
     const { show } = Provider.useModal();
     const { getCFDState } = Provider.useCFDContext();
-    const platform = getCFDState('platform') ?? PlatformDetails.mt5.platform;
+    const platform = getCFDState('platform') ?? PlatformDetails.dxtrade.platform;
```

Comment:
@shontzu-deriv why did you change default platform from MT5 to DXtrade? I see you are trying to make this component common for all CFDs, not just for MT5, but why did you changed default?